tags/file_manager/file_manager.talon

```diff
@@ -8,25 +8,25 @@ manager close: user.file_manager_hide_pickers()
 manager refresh: user.file_manager_update_lists()
 go <user.system_path>: user.file_manager_open_directory(system_path)
 (go parent | daddy): user.file_manager_open_parent()
+^follow {user.file_manager_directories}$:
```

Comment:
Does this conflict with cursorless's "follow" command?

Reply:
I don't believe so as I don't think anything implements cursorless and also the file manager tag
and if they did directories names are going to be different to most cursorless targets

But either way this isn't a new command, just a moved command in the same file

Reply:
Oh I see that now. I misread the diff. Thanks for the clarification.

Reply:
yeah the diff is a bit confusing as the old file is a bit confusing so had to move around a lot of the lines